includes/admin/class-wc-admin-dashboard.php

```diff
@@ -286,6 +291,70 @@ public function recent_reviews() {
 			echo '<p>' . __( 'There are no product reviews yet.', 'woocommerce' ) . '</p>';
 		}
 	}
+
+	/**
+	 * Network orders widget.
+	 */
+	public function network_orders() {
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_script( 'network-orders', WC()->plugin_url() . '/assets/js/admin/network-orders' . $suffix . '.js', array( 'jquery', 'underscore' ), WC_VERSION, true );
+
+		$user = wp_get_current_user();
+		$blogs = get_blogs_of_user( $user->ID );
+		$blog_ids = wp_list_pluck( $blogs, 'userblog_id' );
+
+		wp_localize_script( 'network-orders', 'woocommerce_network_orders', array(
+			'nonce' => wp_create_nonce( 'wp_rest' ),
+			'sites' => array_values( $blog_ids ),
+			'order_endpoint' => get_rest_url( null, 'wc/v2/orders/network' ),
+		) );
+		?>
+		<div class="post-type-shop_order">
+			<div id="woocommerce-network-order-table-loading" class="woocommerce-network-order-table-loading is-active">
+				<p>
+					<span class="spinner is-active"></span> <?php esc_html_e( 'Loading network orders', 'woocommerce' ); ?>
+				</p>
+
+			</div>
+			<table id="woocommerce-network-order-table" class="woocommerce-network-order-table wp-list-table">
+				<thead>
+					<tr>
+						<td><?php esc_html_e( 'Order', 'woocommerce' ); ?></td>
+						<td><?php esc_html_e( 'Status', 'woocommerce' ); ?></td>
+						<td><?php esc_html_e( 'Total', 'woocommerce' ); ?></td>
+					</tr>
+				</thead>
+				<tbody id="network-orders-tbody">
+
+				</tbody>
+			</table>
+			<div id="woocommerce-network-orders-no-orders" class="woocommerce-network-orders-no-orders">
```

Comment:
We can drop the ID if we change up the JS slightly. Asking about that in separate comment.